### DISCUSSION TITLE
Blender to Space Team Pro Error

### DISCUSSION THREAD

**Discussion by acabello7151** [SimDynamX/SpaceTeamsPRO#23] [2023 Mars City Design Competition Q&A]
Good Morning, we are having issues exporting our file from blender to spaceteampro. We have followed the instructions from the video provided, yet it doesn't work. when in blender i make sure i make that file into a glb file, as the video said, but when we do that and move it to the files from spaceteampro, we get a error message on the bottom left.

**Comment by connorjak** [comment]
What error message do you get? Is in an error message in Blender, in Windows, or in the Space Teams Pro app (SpaceCRAFT)?

@AlexSein98 is our 3D model import expert; please weigh in.

**Reply by acabello7151** [reply]
![EE326232-5C89-4794-B8E0-A70521C00909_1_201_a](https://github.com/SimDynamX/SpaceTeamsPro/assets/157191534/8c3b73a7-13e1-4edb-bccf-8c29d1f83efb)

This is what I get on Space Teams Pro

**Reply by connorjak** [reply]
Oh, you can't directly select .glb files in AddEntity to have them be imported. You can instead use the Runtime Loaded Mesh functionality to load that 3D model as configured in an EntityConfig (explained in the 3D model import youtube videos). I see how that could be confusing though; we should probably make Add Entity only show valid EntityConfig files.
